modules/service/src/main/scala/lucuma/odb/logic/TimeEstimateService.scala

```diff
@@ -116,7 +116,7 @@ object TimeEstimateService:
         OptionT.fromOption(m.get(oid)).flatMap: data =>
           OptionT
             .fromOption(data.cachedFullTimeEstimate)  // try the cache first
-            .orElse
+            .orElse:
```

Comment:
This is the bug.  Adding the `:` makes it work as intended and requires fixing the `cachedFullTimeEstimate` value.